lms/static/js/spec/instructor_dashboard/student_admin_spec.js

```diff
@@ -183,7 +234,7 @@ define(['jquery', 'coffee/src/instructor_dashboard/student_admin', 'js/common_he
                 expect(studentadmin.$request_response_error_ee.text()).toEqual(full_error_message);
             });
 
-            it('binds list entrance exam task history ajax call and the result will be success', function() {
+            it('initiates listing of entrance exam task history when button is clicked', function() {
                 studentadmin.$btn_entrance_exam_task_history.click();
                 // expect error to be shown since student identifier is not set
                 expect(studentadmin.$request_response_error_ee.text()).toEqual(gettext("Please enter a student email address or username."));
```

Comment:
@ziafazal -- looks like one of @srpearce's revisions did not make it in yet -- can you please address?  Thanks!